index.js

```diff
@@ -121,22 +121,30 @@ app.listen(3001, () => {
   console.log("Server running on port 3001");
  });
 
+ var receiptCalls = [];
```

Comment:
Not necessary to keep a global reference of results, this could be a source of race conditions